.github/workflows/build.yml

```
@@ -24,6 +24,11 @@ jobs:
     runs-on: ubuntu-22.04
 
     steps:
+    - name: Install bats
+      run: |
+        sudo apt update
+        sudo apt install -y bats
```

```suggestion
        sudo apt-get update
        sudo apt-get install -y bats
```
_Optional_: I know I added this as apt, but since this is a higher impact location it might be preferable to use apt-get